Essentials/src/config.yml

```diff
@@ -726,6 +726,10 @@ min-money: -10000
 # Enable this to log all interactions with trade/buy/sell signs and sell command.
 economy-log-enabled: false
 
+# Enable this to make the economy log favor human readable messages instead of parseable ones.
+# Requires economy-log-enabled to be set to true.
+prettier-economy-log: false
```

Comment:
```suggestion
# Enable this to use a more human-readable format in the economy log.
# WARNING: This may break other plugins or tools that attempt to parse the old economy log format.
# Requires economy-log-enabled to be set to true.
prettier-economy-log: false
```